mmdet3d/datasets/waymo_dataset.py

```diff
@@ -184,6 +183,8 @@ def format_results(self,
                                                   pklfile_prefix,
                                                   submission_prefix)
         if 'waymo' in data_format:
+            from ..core.evaluation.waymo_utils.prediction_kitti_to_waymo import \
+                KITTI2Waymo  # noqa
```

Comment:
is noqa here necessary?

Reply:
Yes. Otherwise flake and isort would have a conflict. (isort tends to put `import` on the previous line while it makes it too long to trigger an error of flake, so I use the # noqa.)